pkg/api/v1alpha1/tinkerbelltemplateconfig_defaults_test.go

```diff
@@ -10,6 +10,7 @@ import (
 
 func TestWithDefaultActionsFromBundle(t *testing.T) {
 	vBundle := givenVersionBundle()
+	diskType := "/dev/sda"
```

Comment:
Should add a test case with nvme as well

Reply:
https://github.com/aws/eks-anywhere/pull/2300#discussion_r889709080